pkg/validate/serialize.go

```diff
@@ -42,24 +42,25 @@ func ValidateCSVManifest(yamlFileName string) error {
 		if errorLog.Errors != nil {
 			fmt.Println()
 			getErrorsFromManifestResult(errorLog.Errors)
-			return fmt.Errorf("Populate all the mandatory fields missing from CSV %s.", csv.GetName())
+			fmt.Printf("Populate all the mandatory fields missing from CSV %s.", csv.GetName())
+			return validator.Error{}
```

Comment:
why are you returning an empty error?

Reply:
Because I'm printing the manifest result in this file itself. The other way can be that I return `errorLog` to the cli and print it from there.

Reply:
This is changed in the next PR I'm about to open so we can discuss this point there.

Reply:
ok looks good then let's merge both